wallet/feepayment.go

```diff
@@ -590,13 +590,14 @@ func (fp *vspFeePayment) confirmPayment() (err error) {
 		return nil
 	case "confirmed":
 		fp.remove("confirmed by VSP")
-		// nothing scheduled
 		fp.mu.Lock()
 		feeHash := fp.feeHash
 		fp.mu.Unlock()
 		err = fp.ticket.UpdateFeeConfirmed(ctx, feeHash, fp.client.URL, fp.client.PubKey)
 		if err != nil {
-			return err
+			fp.client.log.Warnf("Rescheduling status check for %v: db err: %v", fp.ticket, err)
+			fp.schedule("confirm payment", fp.confirmPayment)
```

Comment:
under what situation would updating the db from confirmPayment fail, but where we expect it to somehow work again later when we call confirmPayment at a later point?

Reply:
Disk is full and space becomes available later?